partiql-planner/src/test/kotlin/org/partiql/planner/PlannerErrorReportingTests.kt

```diff
@@ -212,27 +215,7 @@ internal class PlannerErrorReportingTests {
                 """.trimIndent(),
                 true,
                 assertOnProblemCount(0, 0),
-                StaticType.unionOf(StaticType.INT4, StaticType.MISSING)
-            ),
-            TestCase(
-                """
-                    -- both branches are missing, problem
-                    CASE WHEN
-                        1 = 1 THEN MISSING
-                        ELSE MISSING END
-                """.trimIndent(),
-                false,
-                assertOnProblemCount(1, 0),
-            ),
-            TestCase(
-                """
-                    -- both branches are missing, problem
-                    CASE WHEN  
-                        1 = 1 THEN MISSING
-                        ELSE MISSING END
-                """.trimIndent(),
-                true,
-                assertOnProblemCount(0, 1),
```

Comment:
Why are these 2 tests deleted?

Reply:
Makes sense w/ me. I think the original context for the "always returns missing" planning error was for data type mismatches (e.g. adding a string with an int). The literal `MISSING` doesn't denote a data type mismatch so we could say this case isn't actually an error.